db/seeds/filter/jobs_seed.js

```diff
@@ -0,0 +1,46 @@
+const moment = require('moment');
+const day = moment.duration(1, 'd');
+const week = moment.duration(1, 'w');
+const fortnight = moment.duration(2, 'w');
+const month = moment.duration(1, 'M');
+const threeMonths = moment.duration(3, 'M');
+
+exports.seed = function (knex) {
+  return knex('jobs').del() // Deletes ALL existing entries
+    .then(function () { // Inserts seed entries one by one in series
+      return knex('jobs').insert({
```

Comment:
according to: http://knexjs.org/#Builder-insert you should be able to insert list in one go `knex('jobs').insert([ job1, job2 ... ]) `